packages/react-components/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "react-components",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "clean": "shx rm -rf dist/",
+    "precompile": "npm run clean",
+    "compile": "tsc",
+    "dev": "tsc --watch",
+    "lint": "eslint src"
+  },
+  "devDependencies": {
+    "@mui/icons-material": "^5.15.10",
+    "@mui/material": "^5.15.10",
+    "@pagopa/pagopa-editorial-components": "^3.1.2",
```

Comment:
The `"@pagopa/pagopa-editorial-components": "^3.1.2",` is not used. What do you think about removing it?

```suggestion
```